scripts/db/applications.sql

```diff
@@ -1,9 +1,15 @@
+CREATE TYPE application_status AS ENUM ('approved', 'rejected', 'pending');
+
 CREATE TABLE applications (
     id SERIAL PRIMARY KEY, 
     name VARCHAR(255) NOT NULL,
     email VARCHAR(255) UNIQUE NOT NULL,
     about TEXT NOT NULL,
     video_link VARCHAR(255),
-    discord_nickname VARCHAR(255) UNIQUE NOT NULL,                                           
-    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP  
-); 
\ No newline at end of file
+    discord_nickname VARCHAR(255) UNIQUE NOT NULL,
+    status application_status DEFAULT 'pending',
+    comment TEXT,
+    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    modified_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    modified_by INTEGER
+);
```

Comment:
We might not notice this now, but later on this might have an impact on performance and memory usage
Defining columns with 255 characters for things that can't realistically be more than 50-60 characters: 
discord_nickname, name, email. 
It's a waste of space and with more and more data we might notice the performance overhead. 
it also concerns other current tables that we have now

Reply:
is it referred to the discord_nickname? I can put it at VARCHAR(50), though the rest are in line with users table

Reply:
i feel like github cut me off mid sentence for some reason but I meant to say that i'm speaking just in general without attachment to the PR  😄 that we need to change most of the varchar(255) fields in the current database, but if you change the discord_nickname in the current PR then it would be lovely ❤️

Reply:
Ah yes, we can do this as part of separate PR / ticket? :)